### PR TITLE
Add splash screen

### PR DESCRIPTION
**Pull request by kamil-rusin**
Waiting for #13 to be approved and then merged into this branch.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
     "react-native-reanimated": "^1.7.0",
     "react-native-safe-area-context": "^0.7.3",
     "react-native-screens": "^2.2.0",
-    "react-native-vector-icons": "^6.6.0",
+    "react-native-splash-screen": "^3.2.0",
+    "react-native-vector-icons": "^6.6.0W",
```
Comment (Lukkub) [inline package.json]: There is no such version on NPM ---> 6.6.0W 
Why you change it ?

File: src/components/PicturesListContainer.js [hunk 2]
```diff
@@ -33,6 +34,7 @@ const PictureListContainer = (props) => {
     }, [dispatch]);
 
     useEffect(() => {
+        SplashScreen.hide();
```
Comment (Lukkub) [inline src/components/PicturesListContainer.js]: Move hide logic to the main App.js file

File: package.json (hunk 1 above)
Comment (kamil-rusin) [reply package.json]: Missclick 😓

File: src/screens/App.js [hunk 3]
```diff
@@ -17,6 +18,8 @@ const WebContentScreen = ({ navigation, route }) => {
 const Stack = createStackNavigator();
 
 const App: () => React$Node = () => {
+    SplashScreen.hide();
```
Comment (Lukkub) [inline src/screens/App.js]: Useeffect

**Pull request merged**